_sass/layout.scss

```diff
@@ -72,36 +74,56 @@
 		grid-column-gap: 0px;
 		grid-template-columns: 1;
 		grid-template-rows: 115px auto auto auto auto;
-		grid-template-areas: 
+		grid-template-areas:
 			"h h h h h h h h h h h h"
 			"i i i i i i i i i i i i"
 			"p p p p p p p p p p p p"
-		    "a a a a a a a a a a a a"
+			"a a a a a a a a a a a a"
 			"c c c c c c c c c c c c"
 			"f f f f f f f f f f f f";
 	}
 }
 
 .page {
-	margin-top: 30px;
+	// margin-top: 30px;
```

Comment:
You're right that it looks better without, but let's just delete the line entirely instead of commenting it out:

```suggestion
```